device/cuda/src/seeding/track_params_estimation.cu

```diff
@@ -0,0 +1,80 @@
+/** TRACCC library, part of the ACTS project (R&D line)
+ *
+ * (c) 2021 CERN for the benefit of the ACTS project
+ *
+ * Mozilla Public License Version 2.0
+ */
+
+#include <cuda/seeding/track_params_estimation.hpp>
+#include <cuda/utils/definitions.hpp>
+
+namespace traccc {
+namespace cuda {
+
+/// Forward declaration of track parameter estimating kernel
+/// The bound track parameters at the bottom spacepoints are obtained
+///
+/// @param seeds_view seeds found by seed finding
+/// @param params_view vector of bound track parameters at the bottom
+/// spacepoints
+__global__ void track_params_estimating_kernel(
+    seed_container_view seeds_view,
+    vecmem::data::vector_view<bound_track_parameters> params_view);
+
+track_params_estimation::output_type track_params_estimation::operator()(
+    host_seed_container&& seeds) const {
+
+    auto n_seeds = seeds.get_headers()[0];
+    output_type params(n_seeds, &m_mr.get());
+
+    auto seeds_view = get_data(seeds, &m_mr.get());
+    auto params_view = vecmem::get_data(params);
+
+    // -- Num threads
+    // The dimension of block is the integer multiple of WARP_SIZE (=32)
+    unsigned int num_threads = WARP_SIZE * 2;
+
+    // -- Num blocks
+    // The dimension of grid is number_of_seeds / num_threads + 1
+    unsigned int num_blocks = seeds.get_headers()[0] / num_threads + 1;
```

Comment:
Very minor, but this has a chance of creating grids that are larger than necessary. Perhaps try:

```suggestion
    unsigned int num_blocks = (seeds.get_headers()[0] + num_threads - 1) / num_threads;
```

Reply:
makes sense. Will fix it